src/cartesi-machine.lua

```diff
@@ -176,69 +173,42 @@ where options are:
 
     --rollup-rx-buffer=start:0x60000000,length:2<<20
     --rollup-tx-buffer=start:0x60200000,length:2<<20
-    --rollup-input-metadata=start:0x60400000,length:4096
-    --rollup-voucher-hashes=start:0x60600000,length:2<<20
-    --rollup-notice-hashes=start:0x60800000,length:2<<20
+    --htif-yield-manual
+    --htif-yield-automatic
 
   --rollup-advance-state=<key>:<value>[,<key>:<value>[,...]...]
     advances the state of the machine through a number of inputs in an epoch
 
     <key>:<value> is one of
-        epoch_index:<number>
         input:<filename-pattern>
-        input_metadata:<filename-pattern>
         input_index_begin:<number>
         input_index_end:<number>
-        voucher:<filename-pattern>
-        voucher_hashes: <filename>
-        notice:<filename-pattern>
-        notice_hashes: <filename>
-        report:<filename-pattern>
+        output:<filename-pattern>
```

Comment:
I just realized something. We are keeping notices/vouchers separate in the server manager for this release. I think we should also keep them separate here. I know this is sad, but I think it will be less confusing.

Reply:
So sad :cry:, separate: `report`, `notice` and `voucher` then.

Reply:
Yeah. 😭

Reply:
This decision got reverted back to a single `output` of verifiable data, as per [output unification](https://discord.com/channels/600597137524391947/1172230296066871407) discussions.